EIPS/eip-2253.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 2253
+title: wallet_getAddressBook JSON-RPC Method
+author: Loredana Cirstea (@loredanacirstea)
+discussions-to: https://ethereum-magicians.org/t/eip-2253-add-wallet-getaddressbook-json-rpc-method/3592
+status: Draft
+type: Standards Track
+category: ERC
```

Comment:
I think this belongs under Interface where the other RPCs are.

Reply:
I was unsure about that. It makes sense to be in the `Interface` category, but I initially looked at https://eips.ethereum.org/EIPS/eip-2015 as an example for `wallet_` methods and saw it in the `ERC` category.
I will change it.

Reply:
2015 was also mistakenly labeled, see #2260.

Reply:
done.